package.json

```diff
@@ -83,7 +83,7 @@
     },
     "dependencies": {
         "@aics/browsing-context-messaging": "~1.1",
-        "@aics/web-3d-viewer": "^2.9.0",
+        "@aics/web-3d-viewer": "^2.9.2",
```

Comment:
do you have a rough idea of what changed from 2.9.0 to 2.9.2?

Reply:
There's a lot of changes, basically everything that's been merged since May of this year. The main reason I upgraded was to add in the viewer state refactor (so I can add in ViewerStateProvider).

I'll wait to release CFE until after https://github.com/allen-cell-animated/website-3d-cell-viewer/pull/321 is merged and included in a release.